scripts/ebpf_tracing.cmd

```diff
@@ -231,18 +210,30 @@ echo Usage: ebpf_tracing.cmd command /trace_path path [/trace_name name] [/rundo
 echo:
 echo Valid parameters:
 echo:
-echo   <command>                            - (mandatory) Valid values are: [start, stop, periodic]
-echo   /trace_path path                     - (mandatory) Path into which the tracing will be located (creates it if it does not exist).
-echo   /trace_name name                     - Name of the logman trace (Default: "ebpf_diag")
-echo   /rundown_period period               - Period, expressed as (H:mm:ss), for saving and generating a new ETL log, and for generating a WFP state snapshot (Default: 0:35:00).
-echo   /max_file_size_mb size               - Maximum size set for an ETL log (Default: 20).
-echo   /max_committed_folder_size_mb count  - Maximum overall size for (most recent) .ETL files to keep in the main 'trace_path\committed' (Default: 200)
+echo   <command>                                - (mandatory) Valid values are: [start, stop, periodic]
+echo   /trace_path path                         - (mandatory) Path into which the tracing will be located (creates it if it does not exist).
+echo   /trace_name name                         - Name of the logman trace (Default: "ebpf_diag")
+echo   /rundown_period period                   - Period, expressed as (H:mm:ss), for saving and generating a new ETL log, and for generating a WFP state snapshot (Default: 0:35:00).
+echo   /max_file_size_mb size                   - Maximum size set for an ETL log (Default: 20).
+echo   /max_committed_folder_size_mb size       - Maximum overall size for (most recent) .ETL files to keep in the main 'trace_path\committed' (Default: 200)
 echo   /max_committed_rundown_state_files count - Number (most recent) of each type of rundown state file to keep in the main 'trace_path\committed' (Default: 1).
 echo:
-echo Examples (overriding defaults):
+echo Behaviour:
+echo - When called with the 'start' command, it will:
+echo 	- Setup the logman session named as defined in 'trace_name', capping circular-log file size to 'max_file_size_mb', and generating every 'rundown_period'.
+echo    - Configure the WFP/eBPF events to be monitored
+echo    - Start the session within the given 'trace_path' directory.
+echo - When called with the 'stop' command, it will:
+echo 	- Stop then delete the logman session, and delete the 'trace_path' directory.
+echo - When called with the 'periodic' command, it will:
+echo 	- Run 'netsh wfp show state' into the 'trace_path' directory, and if the file is under 'max_file_size_mb', it will move it into the 'trace_path\committed' subfolder, adding a timestamp to its name.
+echo    - Run down the program state using bpftool, to capture the program output: link, map, and map content outputs, and store them in "bpf_state.txt".
```

Comment:
The wfp state description also mentions about moving to committed and adding a timestamp to the file name. the bpftool output does the same. Do we want to add that to the description here as well?

Reply:
agreee, added.